tests/test_operators.py

```diff
@@ -170,6 +171,37 @@ def test_split_by_value(self):
         for input_dict, ouput_dict in zip(inputs[1:], outputs_2):
             self.assertDictEqual(input_dict, ouput_dict)
 
+    def test_merge(self):
+        # Test with default params
+        input_multi_stream = MultiStream(
+            {"test": [{"field": "test1"}], "validation": [{"field": "validation1"}], "train": [{"field": "train1"}]}
+        )
+        output_multi_stream = MergeStreams()(input_multi_stream)
+        self.assertListEqual(list(output_multi_stream.keys()), ["all"])
+        all = list(output_multi_stream["all"])
+        expected_all = [
+            {"field": "test1", "origin": "test"},
+            {"field": "validation1", "origin": "validation"},
+            {"field": "train1", "origin": "train"},
+        ]
+        self.assertEqual(len(all), len(expected_all))
+        for input_dict, output_dict in zip(all, expected_all):
+            self.assertDictEqual(input_dict, output_dict)
+
+        # test with parameters
+        input_multi_stream = MultiStream(
+            {"test": [{"field": "test1"}], "validation": [{"field": "validation1"}], "train": [{"field": "train1"}]}
+        )
+        output_multi_stream = MergeStreams(
+            streams_to_merge=["test", "train"], new_stream_name="merged", add_origin_stream_name=False
+        )(input_multi_stream)
+        self.assertListEqual(list(output_multi_stream.keys()), ["merged"])
```

Comment:
suggestion: this is the same code as above, IIUC, for comparing a stream to expected value. maybe consider a function for that.

Reply:
Done.